boruta/readme.md

```diff
@@ -0,0 +1,30 @@
+# Boruta Feature Selection
+
+In this package you'll find two scripts implementing Boruta Feature Selection:
+
+- [Boruta 1-click Feature Selection](./1-click-boruta)
+  Starting from a dataset, creates a new one removing the unimportant features
+  according to the Boruta Feature Selection algorithm.
+- [Boruta Feature Selection](./boruta)  General Boruta Feature Selection
+  algorithm to rank fields from a dataset into three categories: important,
+  unimportant and undecided.
```

Comment:
I wrote both just following the cross-validation example (I think they are published separately in the gallery) but if you think it's not worth the effort right now, I can remove the boruta folder and leave just the 1-click.